lib/dal-test/src/helpers.rs

```diff
@@ -228,3 +228,20 @@ pub async fn fetch_resource_last_synced_value(
         .await?;
     Ok(last_synced_value)
 }
+
+/// Extracts the value and validation from a raw property edtior value.
+pub fn extract_value_and_validation(
```

Comment:
Moved while authoring a test that I used to find the bug. The test was discarded, but this should be useful for bubbling up errors.